recipes/recipes_emscripten/r-nlme/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# Using flang as a WASM cross-compiler
+# https://github.com/serge-sans-paille/llvm-project/blob/feature/flang-wasm/README.wasm.md
+# https://github.com/conda-forge/flang-feedstock/pull/69
+micromamba install -p $BUILD_PREFIX \
+    conda-forge/label/llvm_rc::libllvm19=19.1.0.rc2 \
+    conda-forge/label/llvm_dev::flang=19.1.0.rc2 \
+    -y --no-channel-priority
+rm $BUILD_PREFIX/bin/clang # links to clang19
+ln -s $BUILD_PREFIX/bin/clang-18 $BUILD_PREFIX/bin/clang # links to emsdk clang
+
+echo "R_HOME=\$(PREFIX)/lib/R" > $BUILD_PREFIX/lib/R/etc/Makeconf
+cat $PREFIX/lib/R/etc/Makeconf >> $BUILD_PREFIX/lib/R/etc/Makeconf
```

Comment:
This is not needed. The activation script of cross-r-base takes care of this.
See https://github.com/conda-forge/r-base-feedstock/blob/7259ce0272e70090e8d08f611995a1d0a71df958/recipe/activate-cross-r-base.sh#L14C2-L15C77